e2e/wdio/core/tests/toolbar.e2e-spec.ts

```diff
@@ -0,0 +1,136 @@
+import { ToolbarPo } from '../pages/toolbar.po';
+import {
+    click, doubleClick, getAttributeByName,
+    getElementArrayLength, getText,
+    isElementClickable, isElementDisplayed,
+
+    refreshPage, scrollIntoView
+} from '../../driver/wdio';
+import {
+    fruitArr, currentDay, date
+} from '../fixtures/appData/toolbar-contents';
+
+describe('Toolbar test suite', function() {
+    const toolbarPage = new ToolbarPo();
+    const {
+        activeInfoToolbar, overflowButton, overflowPriorityButton, moreButton, overflowBody, alwaysButton,
+        overflowGroupingButton, checkbox, dropdownMenu, dropdownOption, inputFieldText, popoverButton, popoverInput,
+        selectedHours, selectedMinutes, navigationUpArrowButton, navigationDownArrowButton, timeItem, period,
+        dayInCalendarButtonByValue, dateTimeButton, okButton, dateTimeInput
+    } = toolbarPage;
+
+    beforeAll(() => {
+        toolbarPage.open();
+    }, 1);
+
+    afterEach(() => {
+        refreshPage();
+    }, 2);
+
+    it('verify info active toolbar is clickable', () => {
+        scrollIntoView(activeInfoToolbar);
+        expect(isElementClickable(activeInfoToolbar)).toBe(true, 'info active toolbar is not clickable');
+    });
+
+    describe('Check Toolbar Overflow example', function() {
+
+        it('verify all buttons are clickable', () => {
+            checkClickableButton(overflowButton);
+        });
+
+        it('verify checkbox', () => {
+            scrollIntoView(checkbox);
+            doubleClick(checkbox);
+            expect(getAttributeByName(checkbox, 'ng-reflect-model')).toBe('true');
```

Comment:
this `ng-reflect-model` is a dev mod attribute, could you please find another way to check if checkbox is marked?